### PR TITLE
Create project Button

### LINKED ISSUE #27: Implementation of "Create Project" button on the projects page
## Description
There is a button currently present in the view, but it is not doing any action.

## Criteria
- [ ] Clicking the button should create a card in the list with default values
- [ ] Should also create the according folder

### PR DESCRIPTION
**Pull request by zelcion**
Closes #27

### CODE REVIEW

File: src/electron/main.ts [hunk 1]
```diff
@@ -27,15 +27,14 @@ export default class Main {
     const __dirname = path.dirname(__filename);
 
     Main.mainWindow = new Main.BrowserWindow({
-      width: 800,
+      width: 1200,
```
Comment (homemmakako) [inline src/electron/main.ts:30]: Nice! Had changed it as well on local branch after the change to auto open the dev tools

File: src/entities/controllers/editor-metadata-controller.ts [hunk 2]
```diff
@@ -8,4 +10,14 @@ export class EditorMetadataController {
...
     return EditorMetadata.fromSerializable(editorData);
   }
+
+  public static async appendProjectIdentifier (identifier : string) : Promise<void> {
+    editorMetadataStoreSingleton.data.update((v) => { v.availableProjectsIds.push(identifier); return v; });
+  }
+
+  public static async saveCurrentMetadata () : Promise<void> {
+    const currentData = get(editorMetadataStoreSingleton.data);
+    console.log(this.fileApi);
```
Comment (homemmakako) [inline src/entities/controllers/editor-metadata-controller.ts:20]: Maybe remove this before merge

File: src/entities/models/project.ts [hunk 3]
```diff
@@ -51,4 +51,15 @@ export class Project implements EditorEntityValue {
       identifier: this.identifier,
     };
   }
+
+  static newEmpty () : Project {
+    return new Project({
+      projectName: "new Project",
+      description: "project description",
```
Comment (homemmakako) [inline src/entities/models/project.ts:58]: Personally I would capitalize the first letter, but thats more of a stylistic choice

**Review by homemmakako — APPROVED** [review]
Some minor comments. Nothing that should block the PR. Good job!

File: src/entities/controllers/editor-metadata-controller.ts (hunk 2 above)
Comment (zelcion) [reply src/entities/controllers/editor-metadata-controller.ts:20]: nice catch

**Pull request merged**